1-exercises/A-array-find/exercise.js

```diff
@@ -17,7 +17,7 @@ let names = [
   "Ahmed",
 ];
 
-let longNameThatStartsWithA = findLongNameThatStartsWithA(names);
+let longNameThatStartsWithA = names.find((name) => name.length > 7);
```

Comment:
The code needs to check 2 criteria and one is missing. A bit touch up is needed. :>